synapse/cores/ram.py

```diff
@@ -38,6 +43,17 @@ def _initCortex(self):
         self.initSizeBy('range', self._sizeByRange)
         self.initRowsBy('range', self._rowsByRange)
 
+        self._blob_store = {'syn:core:type': self._getCoreType()}
+        self._blob_loadbus = EventBus()
```

Comment:
Any reason we need separate save/load bus for blobs? Seems like they could probably just pass across the existing ones.